platform/linuxbsd/export/export_plugin.cpp

```diff
@@ -200,3 +265,185 @@ Error EditorExportPlatformLinuxBSD::fixup_embedded_pck(const String &p_path, int
 	}
 	return OK;
 }
+
+Ref<Texture2D> EditorExportPlatformLinuxBSD::get_run_icon() const {
```

Comment:
I feel a standalone file should be added for this export platform given its getting a bit big

Reply:
Like a separate files for `export` and `run` parts?

Reply:
I think @reduz might have misread the file paths. This is already a dedicated file (but indeed it used to be that both LinuxBSD and Windows were together in `editor/editor_export.cpp`).